File: code/C/Chapter2/2-2/sequence_search.c
#include <stdio.h>

int sequence_search(int *arr, int n, int key) {
    for (int i = 0; i < n; i++) {
        if (arr[i] == key) {
            return i;
        }
    }
    return -1;
}

int main() {
    int arr[] = {40, 9, 20, 93, 7, 34, 85, 91};
    int n = sizeof(arr) / sizeof(arr[0]);
    int key = 34;
    printf("index = %d\n", sequence_search(arr, n, key));
    return 0;
}

File: code/Python/Chapter2/2-1/sequence_search.py
def sequence_search(lst, key):
    for i in range(len(lst)):
        if lst[i] == key:
            return i
    return -1

lst = [40, 9, 20, 93, 7, 34, 85, 91]
key = 34
print("index =", sequence_search(lst, key))